vizro-core/examples/_dev/app.py

```diff
@@ -3,33 +3,298 @@
+    title="Multiple Tabs",
+    components=[
+        vm.Tabs(
+            tabs=[
+                vm.Container(
+                    title="Tab 1 container 1",
+                    components=[
+                        vm.Graph(
+                            figure=px.scatter(
+                                gapminder,
+                                title="Graph_44",
+                                x="gdpPercap",
+                                y="lifeExp",
+                                size="pop",
+                                color="continent",
+                            ),
+                        ),
+                    ],
+                ),
+                vm.Container(
+                    title="Tab 1 container 2",
+                    components=[
+                        vm.Graph(
+                            figure=px.scatter(
+                                gapminder,
+                                title="Graph_441",
+                                x="gdpPercap",
+                                y="lifeExp",
+                                size="pop",
+                                color="continent",
+                            ),
+                        ),
+                    ],
+                ),
+            ],
+        ),
+        vm.Tabs(
+            tabs=[
+                vm.Container(
+                    title="Tab 2 container",
+                    components=[
+                        vm.Graph(
+                            figure=px.scatter(
+                                gapminder,
+                                title="Graph_45",
+                                x="gdpPercap",
+                                y="lifeExp",
+                                size="pop",
+                                color="continent",
+                            ),
+                        ),
+                    ],
+                ),
+                vm.Container(
+                    title="Tab 2 container 2",
+                    components=[
+                        vm.Graph(
+                            figure=px.scatter(
+                                gapminder,
+                                title="Graph_451",
+                                x="gdpPercap",
+                                y="lifeExp",
+                                size="pop",
+                                color="continent",
+                            ),
+                        ),
+                    ],
+                ),
+            ],
+        ),
+    ],
+)
+
+
+tabs_with_nested_containers = vm.Page(
+    title="Tabs with nested containers",
+    components=[
+        vm.Tabs(
+            tabs=[
+                # LN: Does it make more sense to put the outer container title as tab label or the innermost?
+                vm.Container(
+                    title="Tab 1",
+                    components=[
+                        vm.Container(
+                            title="Nested Container Title",
```

Comment:
Not sure if you discussed this before, but with a fresh outside view I think this makes sense as it is. My understanding:
- I put a container in `tabs` of `vm.Tabs`
- the title of that container will become the tab, ie the thing you click on to switch between. The container itself looses its title
- If I put another container in the container, that title is kept, because it is not a "first-level" container

That all makes sense to me as it is. Would be curious what alternative would also be intuitive

Reply:
Good to hear! 😄 I also find the current implementation intuitive, but the other approach we initially discussed is basically the other way around. So the title of the innermost container will be placed inside the tab label, while the title of the outermost containers are shown. I've reversed it in the current implementation, so putting the title of the outermost container inside the tab label and showing the title of the inner containers.

Reply:
Great! That simplifies a lot! 

Yes, I actually didn't think about the siblings case - definitely makes more sense to go with the current implementation then.